client/client.go

```diff
@@ -524,6 +527,44 @@ func (c *client) GetRegion(ctx context.Context, key []byte) (*Region, error) {
 	return c.parseRegionResponse(resp), nil
 }
 
+func (c *client) GetRegionFromMember(ctx context.Context, key []byte, memberURLs []string) (*Region, error) {
+	if span := opentracing.SpanFromContext(ctx); span != nil {
+		span = opentracing.StartSpan("pdclient.GetRegionFromMember", opentracing.ChildOf(span.Context()))
+		defer span.Finish()
+	}
+	start := time.Now()
+	defer func() { cmdDurationGetRegion.Observe(time.Since(start).Seconds()) }()
+
+	var resp *pdpb.GetRegionResponse
+	for _, url := range memberURLs {
```

Comment:
The time of getting regions is decided by the sequence of the member list?

Reply:
Is it going to replace GetRegion? @JmPotato

Reply:
No. We plan to use it with the location labels of TiDB after to provide lower latency, but for now, this is just a new method added.

Reply:
if it is decided by the sequence of the member list, it's may have some latency when the front of this list can not be  reached

Reply:
Yes. But the member list is determined by the user.